Makefile.am

```diff
@@ -199,6 +201,47 @@ cc_oci_runtime_CFLAGS = \
 	$(LIBMNL_CFLAGS) \
 	$(UUID_CFLAGS)
 
+libexec_SCRIPTS = cc-proxy
+
+AM_V_GO    = $(am__v_GO_@AM_V@)
+am__v_GO_  = $(am__v_GO_@AM_DEFAULT_V@)
+am__v_GO_0 = @echo "  GOBUILD " $@;
+am__v_GO_1 =
+
+systemdservice_in_files =		\
+	proxy/cc-proxy.service.in	\
+	proxy/cc-proxy.socket.in
+
+systemdservice_files = 		\
+	proxy/cc-proxy.service	\
+	proxy/cc-proxy.socket
+
+CLEANFILES += $(systemdservice_files)
+
+if HAVE_SYSTEMD
+GENERATED_FILES += $(systemdservice_files)
+systemdservicedir   = $(systemdsystemunitdir)
+systemdservice_DATA = $(systemdservice_files)
+endif
+
+proxy_ldflags = "-X main.socketPath=$(localstatedir)/run/cc-oci-runtime/proxy.sock"
+cc-proxy: $(cc_proxy_sources)
+	$(AM_V_GO)go build -o $@ -ldflags=$(proxy_ldflags) $(srcdir)/proxy
+
+cc_proxy_sources =			\
+	proxy/api/api.go		\
+	proxy/api/fdpassing.go		\
+	proxy/protocol.go		\
+	proxy/proxy.go			\
+	proxy/socket_activation.go	\
+	proxy/syscall.go		\
+	proxy/vm.go
+
+CHECK_DEPS += check-proxy
+
+check-proxy:
+	go test -timeout 2s $(srcdir)/proxy
```

Comment:
Thinking of the Travis vector, maybe run as `go test -v ...` (and possibly redirect to a file for display on failure)?